README.md

```diff
@@ -10,41 +10,79 @@
 
 # Temporary Note
 
-This project is not ready for deployment and the documentation may be incorrect.
+  Changes are made on a daily basis to documentation and files. Documentation may not be accurate due to the rapid changes that are occuring. In its current state, this repo is not ready for deployment.
 
 # Welcome!
 
 This is incoming v2 front-end application for [OperationCode](https://operationcode.org). We highly recommend [joining our organization](https://operationcode.org/join) to receive an invite to our Slack team. From there, you'll want to join the `#oc-projects` channel. You can get help from multiple professional developers, including people who have worked on the application since day 1!
 
-Before contributing, please review our [GitHub Workflow]().
+Before contributing, please review our [Contributing Guide](CONTRIBUTING.md)
 
-## Maintainers
+# Maintainers
```

Comment:
Lets put these back to `##` and add a second bang (#) to Welcome. The number of bangs emulates the HTML `<header>` semantics. As such, there should only ever be one `<h1>` (or `#`, in this case) per page (markdown file, in this case).